tests/integ/test_workflow_with_clarify.py

```diff
@@ -0,0 +1,288 @@
+    )
+
+
+@pytest.fixture(scope="module")
+def data_bias_config():
+    return BiasConfig(
+        label_values_or_threshold=[1],
+        facet_name="F1",
+        facet_values_or_threshold=[0.5],
+        group_name="F2",
+    )
+
+
+@pytest.yield_fixture(scope="module")
+def model_name(sagemaker_session, cpu_instance_type, training_set):
+    job_name = utils.unique_name_from_base("clarify-xgb")
+
+    with timeout.timeout(minutes=integ.TRAINING_DEFAULT_TIMEOUT_MINUTES):
+        ll = LinearLearner(
+            "SageMakerRole",
+            1,
+            cpu_instance_type,
+            predictor_type="binary_classifier",
+            sagemaker_session=sagemaker_session,
+            disable_profiler=True,
+        )
+        ll.binary_classifier_model_selection_criteria = "accuracy"
+        ll.early_stopping_tolerance = 0.0001
+        ll.early_stopping_patience = 3
+        ll.num_models = 1
+        ll.epochs = 1
+        ll.num_calibration_samples = 1
+
+        features, label = training_set
+        ll.fit(
+            ll.record_set(features.astype(np.float32), label.reshape(-1).astype(np.float32)),
+            job_name=job_name,
+        )
+
+    with timeout.timeout_and_delete_endpoint_by_name(job_name, sagemaker_session):
+        ll.deploy(1, cpu_instance_type, endpoint_name=job_name, model_name=job_name, wait=True)
+        yield job_name
+
+
+@pytest.fixture(scope="module")
+def model_config(model_name):
+    return ModelConfig(
+        model_name=model_name,
+        instance_type="ml.c5.xlarge",
+        instance_count=1,
+        accept_type="application/jsonlines",
+    )
+
+
+@pytest.fixture(scope="module")
+def model_predicted_label_config(sagemaker_session, model_name, training_set):
+    predictor = LinearLearnerPredictor(
+        model_name,
+        sagemaker_session=sagemaker_session,
+    )
+    result = predictor.predict(training_set[0].astype(np.float32))
+    predictions = [float(record.label["score"].float32_tensor.values[0]) for record in result]
+    probability_threshold = statistics.median(predictions)
+    return ModelPredictedLabelConfig(label="score", probability_threshold=probability_threshold)
+
+
+def test_workflow_with_clarify(
+    data_config,
+    data_bias_config,
+    model_config,
+    model_predicted_label_config,
+    pipeline_name,
+    role,
+    sagemaker_session,
+):
+
+    instance_type = ParameterString(name="InstanceType", default_value="ml.m5.xlarge")
+    instance_count = ParameterInteger(name="InstanceCount", default_value=1)
+
+    analysis_config = data_config.get_config()
+    analysis_config.update(data_bias_config.get_config())
+    (
+        probability_threshold,
+        predictor_config,
+    ) = model_predicted_label_config.get_predictor_config()
+    predictor_config.update(model_config.get_predictor_config())
+    analysis_config["methods"] = {"pre_training_bias": {"methods": "all"}}
+    analysis_config["methods"] = {"post_training_bias": {"methods": "all"}}
```

Comment:
looks like you are setting the same key here

Reply:
Thanks, setting it just once now